deepchem/feat/graph_features.py

```diff
@@ -263,14 +290,30 @@ class WeaveFeaturizer(Featurizer):
 
   name = ['weave_mol']
 
-  def __init__(self):
+  def __init__(self, graph_distance=True, explicit_H=None):
```

Comment:
I just deleted everything tricky here, so it should be directly depending on the input parameter `explicit_H` now.